.checksyncrc.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "./src/checksync.schema.json",
+    "$comment": "This is the config file for our integration tests. By not specifying it directly, we also test configuration file discovery.",
+    "autoFix": false,
+    "comments": [
+        "//",
+        "#",
+        "{/*"
+    ],
+    "dryRun": false,
+    "excludeGlobs": [
+        "**/excluded/**"
+    ],
+    "ignoreFiles": [
+        "**/ignore-file.txt"
+    ],
+    "json": false
+}
```

Comment:
An example config file for checksync. Note that the schema can be specified at the top of the file as per JSON Schema specifications (the schema is shipped with the tool).

Note that folks don't need to include this - their configs will still be validated against this schema, regardless. However, for tools like VSCode, this tells it what schema to use and provides autocomplete/intellisense accordingly.